src/test/java/ApiTests.java

```diff
@@ -0,0 +1,27 @@
+import io.qameta.allure.Owner;
+import models.Root;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static helpers.ConfigSetUp.getProjectName;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static specs.RestAssuredSpec.requestSpecification;
+import static specs.RestAssuredSpec.resSpec;
+
+public class ApiTests {
+    @Test
+    @DisplayName("Get all permitted dashboard resources for specified project")
+    @Owner("Oleg Chubryk")
+    public void getAllPermittedDashboard() {
```

Comment:
test name about all, in test you call only one

Reply:
it is the description of the method from Swagger
<img width="882" alt="image" src="https://user-images.githubusercontent.com/58031712/194538327-897ffea4-1756-4bf3-92b9-65df0be09052.png">

Reply:
yes but you test not all of them

Reply:
This test is written nominally, just to be. My goal was not to create real test.

Reply:
okay then, I see